metals/src/main/scala/scala/meta/internal/builds/GradleBuildTool.scala

```diff
@@ -21,6 +21,9 @@ case class GradleBuildTool() extends BuildTool {
        |initscript {
        |    repositories {
        |        mavenCentral()
+       |        
+       |        // TODO - Remove
```

Comment:
Will need to update Bloop version and remove additional repos once Bloop is released